src/arrayMethodJoin.js

```diff
@@ -5,8 +5,28 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let result = '';
+
+    if (this.length) {
```

Comment:
```suggestion
    if (!this.length) {
      return '';
    }
```

после этог делаешь все что тебе нужно